### ISSUE TITLE
Incorrect rounding when format 0.14

### ISSUE THREAD

**Issue by smilenovel** [BenjaminVanRyseghem/numbro#375]
numbro version: 2.1.0

value = numbro(0.14).format('0%');
console.log(value); //14.000000000000002%

The expect result is 14%. But I got 14.000000000000002%

**Comment by BenjaminVanRyseghem** [comment]
yep, JS is not very accurate when it comes to float. But since we use BigNumber, we should be able to avoid this.

**Comment by andrewQwer** [comment]
Have the same issue with percentage formatting:
`numbro('0.14').format("0,0%");`
output: `14.000000000000002%`

`numbro('0.14').format("0,0.000000000000000000000000000000%")`
output: `14.000000000000003552713678800501%`

But if original value is too large so thousands separator appears it looks like this:

`numbro('10.14').format("0,0%");`
output: `1,014%`

**Comment by jamesonhill** [comment]
This happens for any multiple of 7 when you do not include the `mantissa` property.

**Comment by andrewQwer** [comment]
Any progress on this?
